SPRACINGH7NANO - First betaflight STM32H750 400Mhz 20x20 mount FC.

hydra opened the pull request:
Website: http://seriouslypro.com/spracingh7nano
Shop: https://shop.seriouslypro.com/sp-racing-h7-nano

`Boards - SPRacingH7NANO.md` to follow in separate commit.

mikeller commented:
Please remember to add a Unified Target configuration for this as required by https://github.com/betaflight/betaflight#new-requirements-for-the-submission-of-new-and-updated-targets

hydra commented:
As noted in #8536, no universal target exists for H750 yet so a config cannot be added.

Suggest merging this as-is until the required unified target has been created, at which point I will be happy to create a new PR for the unified target.

hydra commented:
Documentation added, ready for merge.

mikeller commented on src/main/target/SPRACINGH7NANO/target.c:
```diff
@@ -0,0 +1,60 @@
+/*
```
This is still missing a Unified Target configuration, see https://github.com/betaflight/betaflight#new-requirements-for-the-submission-of-new-and-updated-targets

hydra replied:
impossible to create using the instructions linked - https://github.com/betaflight/betaflight/blob/master/unified_targets/docs/CreatingAUnifiedTarget.md as there is no unified target for any H7 MCU yet.

hydra commented:
ok, after discussion in Slack and on #8536 it was noted that:

* the missing NUCLEOH743 unified target config was an oversight.
* the instructions I was directed at can only be followed up to step 2 at this time.
* the target configuration for the SPRACINGH7NANO can be created and added to this PR but cannot be tested yet.

I've added a unified target config to this PR too.

LGTM!

mikeller commented on src/main/target/SPRACINGH7NANO/SPRACINGH7NANO.mk:
```diff
@@ -0,0 +1,15 @@
+FEATURES       += VCP ONBOARDFLASH
```
This should go into `target.mk`.

hydra replied:
done

mikeller commented on unified_targets/configs/SPRACINGH7NANO.config:
```diff
@@ -0,0 +1,226 @@
+resource MOTOR 6 B07
+resource MOTOR 7 C06
+resource MOTOR 8 C07
+resource SERVO 1 NONE
```
The lines containing `NONE` in here should be removed.

hydra replied:
done

hydra commented:
rebased on master, ready for merge after #8536
